### ISSUE TITLE
Python 3

### ISSUE THREAD

**Issue by CurtisWJohnson** [Azure-Samples/iot-hub-python-raspberrypi-client-app#20]
The python3 command
`python3 app.py '<your Azure IoT hub device connection string>' `
does not work but `python2 app.py '<your Azure IoT hub device connection string>' ` does work.

**Comment by sandeepnegi1996** [comment]
yes you are right in python 3 it is showing import error can anyone rectify it

**Comment by markpatterson27** [comment]
Did you install the python 3 modules and not just the python 2 modules?

i.e. `pip3 install azure-iothub-device-client`, `pip3 install azure-iothub-service-client`.
Or if you're building the C libraries, `sudo ./setup.sh --python-version 3.5`
